scipy/stats/_hypotests.py

```diff
@@ -1225,3 +1226,242 @@ def cramervonmises_2samp(x, y, method='auto'):
+    result : `~scipy.stats._result_classes.TukeyKramerTestResult` instance
+        The return value is an object with the following attributes:
+
+        sig_level : float
+            The desired significance level (copied from `tukeykramer` input).
+        statistic : float ndarray
+            The computed statistic of the test for each comparison. The
+            (i, j) index is the statistic for the ith - jth group comparison.
+        pvalue : float ndarray
+            The associated p-value from the studentized range distribution. The
+            (i, j) index is the p-value for the ith - jth group comparison.
+
+
+        The object has the following methods:
+
+        confidence_ci() :
+            Compute the confidence interval for the significance level
+            specified in the test.
+
+    Notes
+    -----
+    The use of this test relies on several assumptions.
+
+    1. There are equal variances between the samples.
+    2. Each sample is from a normally distributed population.
+
+    References
+    ----------
+    .. [1] NIST/SEMATECH e-Handbook of Statistical Methods, "7.4.7.1. Tukey's
+           Method."
+           https://www.itl.nist.gov/div898/handbook/prc/section4/prc471.htm,
+           28 November 2020.
+    .. [2] Abdi, Herve & Williams, Lynne. (2021). Tukey's Honestly Signiflcant
+           Difierence (HSD) Test.
+    .. [3] "One-Way ANOVA Using SAS PROC ANOVA & PROC GLM." SAS
+           Tutorials, 2007, www.stattutorials.com/SAS/TUTORIAL-PROC-GLM.htm.
+    .. [4] Kramer, Clyde Young. "Extension of Multiple Range Tests to Group
+           Means with Unequal Numbers of Replications." Biometrics, vol. 12,
+           no. 3, 1956, pp. 307-310. JSTOR, www.jstor.org/stable/3001469.
+           Accessed 25 May 2021.
+
+
+    Examples
+    --------
+    Here are some data comparing the time to relief of three brands of
+    headache medicine, reported in minutes. Data adapted from [3]_.
+
+    >>> from scipy.stats import tukeykramer
+    >>> group0 = [24.5, 23.5, 26.4, 27.1, 29.9]
+    >>> group1 = [28.4, 34.2, 29.5, 32.2, 30.1]
+    >>> group2 = [26.1, 28.3, 24.3, 26.2, 27.8]
+
+    We would like to see if the means between any of the groups are
+    significantly different. First, visually examine a box and whisker plot.
+
+    >>> import matplotlib.pyplot as plt
+    >>> fig, ax = plt.subplots(1, 1)
+    >>> ax.boxplot([group0, group1, group2])
+    >>> ax.set_xticklabels(["group0", "group1", "group2"])
+    >>> ax.set_ylabel("mean")
+    >>> plt.show()
+
+    From the box and whisker plot we can see overlap in the interquartile
+    ranges group 1 to group 2 and group 3, but we can apply the ``tukeykramer``
+    test to determine if the difference between means is significant.
+
+    >>> res = tukeykramer(group0, group1, group2)
+    >>> for ((i, j), p) in np.ndenumerate(res.pvalue):
+    >>>     # filter out self comparisons
+    >>>     if i != j:
+    >>>         print(f"({i} - {j}) {p:.03f}")
```

Comment:
```suggestion
    ...     # filter out self comparisons
    ...     if i != j:
    ...         print(f"({i} - {j}) {p:.03f}")
```

Reply:
I'm not sure if these (below, too) will fix all the refguide errors, but it is at least part of the problem.